frontend/packages/console-app/src/components/quick-starts/catalog/QuickStartTileHeader.tsx

```diff
@@ -29,14 +29,17 @@ const QuickStartTileHeader: React.FC<QuickStartTileHeaderProps> = ({ status, dur
 
   return (
     <div className="co-quick-start-tile-header">
-      <Title headingLevel="h3">{name}</Title>
+      <Title headingLevel="h3" data-test={`tile-${name}`}>
+        {name}
+      </Title>
       <div className="co-quick-start-tile-header__status">
         {status !== QuickStartStatus.NOT_STARTED && (
           <Label
             className="co-quick-start-tile-header--margin"
             variant="outline"
             color={statusColorMap[status]}
             icon={<StatusIcon status={status} />}
+            data-test="quick-start-card-status"
```

Comment:
Based on the context we know that we are in a quick start card (or tile)

```suggestion
            data-test="status"
```

Reply:
Updated